server/src/core/server/graph/mutators/Comments.ts

```diff
@@ -254,6 +257,12 @@ export const Comments = (ctx: GraphContext) => ({
     // Publish that the comment was featured.
     await publishCommentFeatured(ctx.broker, comment);
 
+    await ctx.notifications.create(ctx.tenant.id, ctx.tenant.locale, {
+      targetUserID: comment.authorID!,
+      comment,
+      type: NotificationType.COMMENT_FEATURED,
+    });
+
```

Comment:
@marcushaddon  and @kabeaty, this is what it looks like to publish a notification for a user. All you need is access to the `ctx.notifications` which is available for every query and mutation in Coral's API.

The content of the notification is also malleable and has many optional fields. It can also be extended to provide new interface types for various kinds of notifications. It's all very flexible to whatever needs to be inside the notification.

As well, there is a convenient `NotificationType` to categorize the notifications and is infinite in how many kinds it can be, but adds a type that makes it easy to create/sort/filter notifications in the future.

This is what I mean by a `one-liner` to publish a notification anywhere. This isn't relevant for either of your DSA work, but it shows how minimal it will be to send new notifications throughout both of your's code.